config/v2/config.go

```diff
@@ -8,101 +8,116 @@ import (
 
 	v1 "github.com/chanzuckerberg/fogg/config/v1"
 	"github.com/chanzuckerberg/fogg/errs"
+	"gopkg.in/yaml.v2"
 )
 
 func ReadConfig(b []byte) (*Config, error) {
 	c := &Config{
 		Docker: false,
 	}
-	e := json.Unmarshal(b, c)
+	e := yaml.Unmarshal(b, c)
+	fmt.Println(string(b))
+	if e != nil {
+		return nil, errs.WrapUser(e, "unable to parse yaml config file")
+	}
+	return c, nil
```

Comment:
you can just `return c, errs.WrapUser(e, ...)`. to avoid having that extra conditional. more conditionals == more complexity == more bugs == need to write more tests